lib/show.js

```diff
@@ -11,7 +11,7 @@ var utils = require("./utils.js");
   - `show.def(x : a): string`
 */
 function showDef(obj) {
-  return "" + obj;
+  return JSON.stringify(obj);
```

Comment:
I don't have a good answer for you.  I'm only just barely getting started in `jsverify`.